projects/storefrontlib/src/cms-components/myaccount/consent-management/components/consent-management.component.html

```diff
@@ -10,18 +10,20 @@
     <ng-container *ngIf="templateList$ | async as templateList">
       <div class="cx-consent-action-links">
         <div class="col-sm-12 col-md-8 col-lg-6">
-          <a
+          <button
```

Comment:
1. when changing to `<button>` element, we should probably add `reset-button` mixin to it's CSS
2. please check if unit tests (and e2e also) were using a selector with `<a>` - in this case, you have to update the tests' selectors

Reply:
1. Sadly, I wasn't able to find any place in our code of usage of such mixin. Am I missing something?
2. Unit tests work correctly